frame/balances/src/migration.rs

```diff
@@ -69,3 +69,29 @@ impl<T: Config<I>, A: Get<Vec<T::AccountId>>, I: 'static> OnRuntimeUpgrade
 		migrate_v0_to_v1::<T, I>(&A::get())
 	}
 }
+
+pub struct ResetInactive<T, I = ()>(PhantomData<(T, I)>);
+impl<T: Config<I>, I: 'static> OnRuntimeUpgrade for ResetInactive<T, I> {
+	fn on_runtime_upgrade() -> Weight {
+		let onchain_version = Pallet::<T, I>::on_chain_storage_version();
+
+		if onchain_version == 1 {
+			// Remove the old `StorageVersion` type.
+			frame_support::storage::unhashed::kill(&frame_support::storage::storage_prefix(
+				Pallet::<T, I>::name().as_bytes(),
+				"StorageVersion".as_bytes(),
+			));
```

Comment:
```suggestion
```
Should have a proper storage version now since it is at version `1`.

Reply:
This is also wrong. The key is not StorageVersion

Reply:
This was old code. AFAIK it should do no harm.